cypress/e2e/2-advanced-examples/files.cy.js

```diff
@@ -51,7 +51,7 @@ context('Files', () => {
 
     // You can read a file and yield its contents
     // The filePath is relative to your project's root.
-    cy.readFile('cypress.config.js').then((json) => {
+    cy.readFile(Cypress.config("configFile")).then((json) => {
```

Comment:
This is no longer `json`

Reply:
Thanks! [93c18bd](https://github.com/cypress-io/cypress-example-kitchensink/pull/551/commits/93c18bd7bbfaec973e444bcc35fa457f598b7187)